src/components/features/Deals/Deals.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Deals.module.scss';
+
+class Deals extends React.Component {
+  render() {
+    const { dealsBig, dealsSmallUp, dealsSmallDown } = this.props;
+
+    return (
+      <div className={styles.root}>
+        <div className='container'>
+          <div className={`row ${styles.dealsBox}`}>
+            <div className={`col-6 ${styles.dealsBig}`}>
+              {dealsBig.map(item => (
+                <div key={item.id} className={styles.dealsBigBox}>
+                  <img
+                    className={styles.photoBig}
+                    alt='product_deals_big'
+                    src={item.photoBackground}
+                  ></img>
+                </div>
+              ))}
+            </div>
+            <div className={`col-6 ${styles.dealsSmall}`}>
+              <div className={`row ${styles.dealsRow}`}>
```

Comment:
Zobacz inspektorem czy to działa? Bo pojawia sie undefined zamiast klasy, literówka?

Reply:
masz rację, poprawiłem

Reply:
![image](https://user-images.githubusercontent.com/26415481/116695464-f427dc00-a9c0-11eb-9087-b5d9ee28bec1.png)

Reply:
poprawione